Add Uptime monitoring content (#162)
elastic/observability-docs: docs/en/observability/configure-uptime-settings.asciidoc

. To access this page, go to *Observability > Uptime*.
. Click *Settings*.
+
[IMPORTANT]
=====
To modify items on this page, you must enable {heartbeat-ref}/securing-heartbeat.html[`all`]
permission.
=====

[[configure-uptime-indices]]
== Configure indices

[NOTE]
=====
The pattern set here only restricts what the {uptime-app} displays. You can still query {es} for
data outside of this pattern.
=====

If your organization has one team for internal IT services, and another
for public services, with each group operating independently, you may want separate {heartbeat} instances per team.

For example, you could create an `it-heartbeat-\*` index pattern and an `external-heartbeat-\*` index pattern.
For each index pattern, you can then create separate roles and users in {es}, each with access to their own spaces,
name them `it` and `external` respectively. For each space, you would set the correct index pattern to match only
the indices that space is allowed to access.

[role="screenshot"]
image::images/heartbeat-indices.png[Heartbeat indices]

[[configure-uptime-alert-connectors]]
== Configure alert connectors

> Alert connectors can be created by clicking the *Create connector* button, which will guide you through selecting the type of

Reviewer: I'd start by explaining what an alert connector is.

> Alert connectors can be created by clicking the *Create connector* button, which will guide you through selecting the type of
> connector and configuring its properties. Once you create a connector it is made available to you anytime you set
> up an alert action in the current space.

Reviewer: ```suggestion
Click *Create connector* and follow the prompts to select a connector type and configure its properties.
After you create a connector, it's available to you anytime you set up an alert action in the current space.
```

Reviewer: Avoid passive voice.

For more information about each connector, see {kibana-ref}/action-types.html[action types and connectors].

[[configure-cert-thresholds]]
== Configure certificate thresholds

> You can modify certificate thresholds to control how Uptime visualizes your TLS values in

Reviewer: ```suggestion
You can modify certificate thresholds to control how Uptime displays your TLS values in
```

Reviewer: I think it's OK to talk about users visualizing things, but I'd avoid anthropomorphizing the software.
